webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/angular/ruleDirectives.js

```diff
@@ -37,35 +37,35 @@
 
 var ruleDirectives = angular.module('ruleDirectives', []);
 
-ruleDirectives.controller('DirectiveCtrl', ['$scope', function($scope) {
+ruleDirectives.controller('DirectiveCtrl', ['$scope', '$timeout', function($scope, $timeout) {
     $scope.directives = {};
-    
-    //really, angular... 
-    //a call to sortDirectives is needed. 
+
+    //really, angular...
+    //a call to sortDirectives is needed.
     //the array here is ONLY used as a cache so
     //that angular can know when things don't change.
     //else, we get infinite looping....
     $scope.sortedDirectives = [];
-    
+
     //needed to correctly display directive by alphaNum
-    $scope.sortDirectives = function() { 
+    $scope.sortDirectives = function() {
       var arr = [];
-      for (var o in $scope.directives) {   
-        arr.push({"id":o, "name": $scope.directives[o] });
-      } 
+      for (var o in $scope.directives) {
```

Comment:
It seems that that part is not working as expected. Directive can't be selected / unselected (groups work)